src/elastio-snap.c

```diff
@@ -1121,7 +1121,16 @@ static MRF_RETURN_TYPE snap_mrf(struct bio *bio);
 #endif
 
 static const struct block_device_operations snap_ops = {
-	.owner = THIS_MODULE,
+	/**
```

Comment:
Don't you need initialize `owner` with `NULL`?

Reply:
No, it's already NULL by default (a static variable)

Reply:
Ha, it says a lot about my `C` knowledge :)